### DISCUSSION TITLE
Import images larger than 980 x 720

### DISCUSSION THREAD

**Discussion by Only-a-cat** [ScratchAddons/ScratchAddons#2085] [Ideas]
maybe add an option to import images larger than 980x720 making them imported with the correct size, instead of being resized to fit the screen

**Comment by cobaltt7** [comment]
But it would be difficult to edit them then. I would just edit them in a dedicated image editor before importing them.
________________________________
From: Only-a-cat ***@***.***>
Sent: Monday, April 5, 2021 7:58:33 PM
To: ScratchAddons/ScratchAddons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ScratchAddons/ScratchAddons] Import images larger than 980 x 720 (#2085)


maybe add an option to import images larger than 980x720 making them imported with the correct size, instead of being resized to fit the screen

**Comment by CST1229** [comment]
Maybe the addon would allow you to change the bitmap canvas to larger resolutions like larger resolutions on turbowarp, but that would mean the image would be cropped if you edit it on Scratch.

**Comment by Illodrus** [comment]
This could work if it forced an import to the vector editor which allows a larger canvas size, especially if the image could be coaxed into SVG format
